OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -127,6 +128,14 @@ public void InvokeCommand(string name, string arg)
 
 					break;
 
+				case "playerxp":
+					var xp = 0;
```

Comment:
We don't use `xp` anywhere else in the code, so please change this to `var experience` and also `
`var experience` and also use `DevPlayerExperience` and `playerexperience` for the order and command names.